whois-commons/src/main/java/net/ripe/db/whois/common/DateTimeProvider.java

```diff
@@ -7,14 +7,14 @@
 
 public interface DateTimeProvider {
 
-    /** Return the current date */
-    LocalDate getCurrentDate();
+    /** Return the current date time in Amsterdam  */
+    LocalDateTime getDateTimeAms();
```

Comment:
Maybe use Cest to make it 100% crystal clear? Often people say CET/Amsterdam interchangably but it really isn't..

Reply:
But should it be CET or CEST? Could not decide which it should be hence I went with Amsterdam Time. Not sure what the general convention is...

Reply:
Please disregard my previous comment.. reading up on it using Ams definitely is more clear than Cet/Cest as that depends on the time of year ;)